src/main/java/com/sourcegraph/cody/SubscriptionTabUtils.kt

```diff
@@ -22,8 +22,9 @@ fun fetchSubscriptionPanelData(
   if (activeAccountType != null) {
     ensureUserIdMatchInAgent(activeAccountType.id, server)
     return if (activeAccountType.isDotcomAccount()) {
-      val codyProFeatureFlag = server.evaluateFeatureFlag(GetFeatureFlag("CodyProJetBrains"))
-      if (codyProFeatureFlag.get() != null && codyProFeatureFlag.get()!!) {
+      val codyProFeatureFlag =
+          server.evaluateFeatureFlag(GetFeatureFlag.CodyProJetBrains).thenApply { it == true }.get()
```

Comment:
Minor nitpick, but I think this is cleaner (less code):
`server.evaluateFeatureFlag(GetFeatureFlag.CodyProJetBrains).get() == true`